### ISSUE TITLE
`global.assert = require('assert')` problem

### ISSUE THREAD

**Issue by atian25** [power-assert-js/power-assert#83]
```
global.assert = require('assert');
describe('test/showcase.test.js', () => {
  const arr = [ 1, 2, 3 ];

  it('power-assert', () => {
    assert(arr[1] === 10);
  });
});
```

then the unit test passed...

**Comment by twada** [comment]
@atian25 Thank you for reporting!
Ah, false positives... I have to wrestle with this.

If you rely on global `assert` object, please use power-assert for the `global.assert` for a while.
([Your temporary fix is right](https://github.com/eggjs/egg-boilerplate-simple/pull/27))

Thanks.

**Comment by atian25** [comment]
hey, @twada any good news on this?

**Comment by twada** [comment]
@atian25 Thanks for pinging me.
Planning. This issue will be fixed by power-assert 2.0.0 (since the fix causes breaking change).

**Comment by atian25** [comment]
@twada  great, any roadmap for 2.0 ?
